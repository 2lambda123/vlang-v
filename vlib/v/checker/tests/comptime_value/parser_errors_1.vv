const my_f32 = $compile_value('my_f32', f32(42.0))
