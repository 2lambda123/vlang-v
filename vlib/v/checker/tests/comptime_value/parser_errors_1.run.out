vlib/v/checker/tests/comptime_value/parser_errors_1.vv:1:16: error: -d values can only be pure literals
    1 | const my_f32 = $d('my_f32', f32(42.0))
      |                ~~~~~~~~~~~~~~~~~~~~~~~
